groups/sig-autoscaling/groups.yaml

```diff
@@ -39,6 +39,46 @@ groups:
       - guyjtempleton@googlemail.com
       - kgolab@google.com
 
+  - email-id: k8s-infra-staging-cluster-autoscaler@kubernetes.io
+    name: k8s-infra-staging-cluster-autoscaler
+    description: |-
+      ACL for staging cluster-autoscaler
+    settings:
+      ReconcileMembers: "true"
+    members:
+      - mwielgus@google.com
+      - lukaszos@google.com
```

Comment:
Let's do a bit of a cleanup. Please remove people who move to some other company or team and are unlikely to do the builds in the near future:

 * lukaszos@google.com 
 * vivek.bagade92@gmail.com
 * aleksandram@google.com

Reply:
I've made the assumption that `lukaszos@google.com` == `losipiuk`, let me know if that's not correct.

Reply:
You are correct.